inst/extdata/parameters/report_GENERAL_2023Q4.json

```
@@ -0,0 +1,8 @@
+{
+  "reporting": {
+    "scenarioOther": "GECO2022_1.5C",
+    "selectScenario": "WEO2022_NZE_2050"
```

```suggestion
    "scenarioOther": "WEO2023_NZE_2050",
    "selectScenario": "WEO2023_NZE_2050"
```

In line with: https://github.com/RMI-PACTA/workflow.transition.monitor/blob/5eedeefc6ff886722033944818fe70e6a066f8f6/parameter_files/ProjectParameters_PA2024CH.yml#L14